client/cli/src/params/offchain_worker_params.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+
+
+//! Offchain worker related configuration parameters.
+//!
+//! A subset of configuration parameters which are relevant to
+//! the inner working of offchain workers. The usage is soley
+//! targetd at handling input parameter parsing providing
+//! a reasonable abstraction.
+
+use structopt::StructOpt;
+use sc_service::Configuration;
+
+use sc_service::config::OffchainWorkerConfig;
+use structopt::clap::arg_enum;
+
+
+use crate::error;
+
+arg_enum! {
+	/// Whether off-chain workers are enabled or not as a whole.
```

Comment:
```suggestion
	/// Whether offchain workers are enabled or not as a whole.
```
Let's keep it consistent at least in this file :)